4.23/ExampleProject/Plugins/PlayFab/Source/PlayFabCommon/Private/PlayFabCommon.cpp

```diff
@@ -27,32 +27,20 @@ class FPlayFabCommonModule : public IPlayFabCommonModuleInterface
     void RegisterSettings();
     void UnregisterSettings();
 
-    /** Callback for when the settings were saved. */
-    bool HandleSettingsSaved();
```

Comment:
seems like we remove a feature with this, but I guess it also decreases complexity if it wasn't being used (It looks like it is not labeled public, so I'm assuming it is not customer facing)